firmware_mod/scripts/PTZpresets.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/sh
+
+################################################
+# Created by Nero                              #
+# neroxps@gmail.com | 2018-5-28 | v0.0.6 Beta  #
+################################################
+
+source /system/sdcard/scripts/common_functions.sh
+
+MOTOR=/system/sdcard/bin/motor
+pid=$$
+echo $pid > /run/PTZ_$pid.pid
+
+# set log
+if [[ -z $3 ]]; then
+    LOG=false
+else
+    LOG=ture
```

Comment:
typo

Reply:
😂
I have found this error.

I have found that there is still a problem with the motor command, so I have not yet pushed to github.